BeeSwift/TimerViewController.swift

```diff
@@ -199,3 +211,19 @@ class TimerViewController: UIViewController {
         }
     }
 }
+
+private extension TimerViewController {
+    static private func timerUnit(goal: Goal) -> TimerUnit {
```

Comment:
Random musing:
Another option for representation could be
1) Make TimerUnit only have actual time units:
```
private enum TimerUnit {
        case hours, minutes
}
```

2) Have this function return an optional result 
```
static private func timerUnit(goal: Goal) -> TimerUnit?
```

3) Have the call site fall back to hours:

```
self.units = Self.timerUnit(goal: goal) ?? .hours
```

This might reduce the number of places in the code which need to understand to treat `other` and `hours` the same.